Upstatement/skela-wp-theme
# Upgrade Docker dependencies

## Changes

**Merge #39 first**

- Update base PHP image ([richarvey/nginx-php-fpm 1.9.0](https://gitlab.com/ric_harvey/nginx-php-fpm/blob/master/docs/versioning.md#php-74), PHP 7.4)
- Switch pager from `more` to `less`
- Update install script to bypass cache on `docker build` so that we don't have to run `wp core update`
- Move docker configuration and scripts into a `docker` directory so that the deploy script doesn't get run on install
- Add server time zone configuration via `TZ` environment variable

(image of expected results)

## How To Test

1. Run `./bin/install` and `./bin/start`
2. Confirm that PHP 7.4.2 is running (`docker-compose exec wordpress php --version`)
3. Confirm that the latest version of WordPress (5.3.2) is installed (`./bin/wp core version`)
4. Confirm that the pager works for CLI commands (e.g. `./bin/wp help`)
